gtsam/linear/tests/testGaussianBayesNet.cpp

```diff
@@ -26,6 +26,8 @@
 #include <boost/assign/list_of.hpp>
 #include <boost/assign/std/list.hpp> // for operator +=
 using namespace boost::assign;
+#include <boost/bind/bind.hpp>
+using namespace boost::placeholders;
```

Comment:
formatting?

Reply:
I believe this is formatted in line with the existing code.

Reply:
Maybe add a suggestion?

Reply:
In other test files, the includes and the usings are grouped together.

Reply:
I think it is appropriate to follow local formatting more than global formatting.
That can be left for another commit/PR.